File: lib_common/src/main/java/com/wujia/lib_common/base/RootResponse.java
package com.wujia.lib_common.base;

import android.text.TextUtils;

import com.wujia.lib_common.data.network.BaseResponse;

import java.io.Serializable;

public class RootResponse extends BaseResponse implements Serializable {
//    public String errcode;
    public String code;
    public String msg;
    public String message;

    @Override
    public boolean isSuccess() {
        return  Constants.HTTP_SUCESS.equals(code) ;
    }

    @Override
    public String getResponseCode() {
        return code;
    }

    @Override
    public String getErrorMsg() {
        return msg;
    }
}

File: lib_common/src/main/java/com/wujia/lib_common/base/RootResponse.kt
package com.wujia.lib_common.base

import com.wujia.lib_common.data.network.BaseResponse

import java.io.Serializable

open class RootResponse : BaseResponse(), Serializable {
    private val msg: String?=null
    var code: String?=null

    //    public String errcode;
    override val responseCode: String?
        get() = code
    override val errorMsg: String?
        get() = msg
    var message: String? = null

    override val isSuccess: Boolean
        get() = Constants.HTTP_SUCESS == responseCode
}
